### PR TITLE
[intro_supply_demand] Chinese in plot

### PR DESCRIPTION
**Pull request by longye-tian**
Hi Matt @mmcky ,

This pull request is to fix the chinese display issue #30  in plot using method in #5 .

Best,
Longye

### CODE REVIEW

File: lectures/intro_supply_demand.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ kernelspec:
 import numpy as np
 import matplotlib.pyplot as plt
 from collections import namedtuple
+from matplotlib import font_manager
+import matplotlib.font_manager
+
+# List all available font names
+font_list = [f.name for f in matplotlib.font_manager.fontManager.ttflist]
+print(font_list)
```
Comment (mmcky) [inline lectures/intro_supply_demand.md:61]: @longye-tian do we need these lines and information?

Comment (longye-tian) [reply lectures/intro_supply_demand.md:61]: Hi Matt @mmcky ,

This is just to know what font are available. So they are temporary, just for me to fix things later. So they will be deleted at the end.

I'm still trying to workout a solution for this problem.

I'll let the team know if I fix it.

Best,
Longye

Comment (mmcky) [reply lectures/intro_supply_demand.md:61]: ah sorry - thanks @longye-tian I thought this was ready. My mistake. Take your time. Thanks for your work.

**Comment by HumphreyYang** [comment]
Hi guys, 

Please kindly see my comments here: https://github.com/QuantEcon/lecture-intro.zh-cn/issues/5#issuecomment-2366775620

I managed to make Chinese characters show on my Linux machine which might cause the same issue on the Linux server.

**Comment by mmcky** [comment]
@longye-tian @HumphreyYang I'm not sure what is needed here. Can you let me know if I need to make some changes on the infrastructure side?

**Comment by HumphreyYang** [comment]
> @longye-tian @HumphreyYang I'm not sure what is needed here. Can you let me know if I need to make some changes on the infrastructure side?

Hi @mmcky,

Nothing needs to be done now. It was my old solution to show Chinese in the matplotlib : )

**Comment by mmcky** [comment]
thanks @longye-tian

**Pull request merged**